### PR TITLE
chore: merge beta into master A18S1

### PR DESCRIPTION
**Pull request by mcdurdin**
@keymanapp-test-bot skip

### CODE REVIEW

File: HISTORY.md [hunk 1]
```diff
@@ -108,6 +108,25 @@
 * chore(common): move to 18.0 alpha (#10713)
 * chore: move to 18.0 alpha
 
+## 17.0.317 beta 2024-05-01
+
+*  (#11322)
+*  (#11321)
```
Comment (jahorton) [inline HISTORY.md:114]:
```suggestion
* chore(web): removes old reference-doc from alpha that has completed its purpose (#11322)
* fix(web): gesture-model initial-state, callback failure handling (#11321)
```

**Review by jahorton — APPROVED** [review]
I don't see any entries for the stuff newly-merged shortly before this PR.  Will we need another pass to handle that, or will it come out in the history-wash on release?

**Comment by mcdurdin** [comment]
> I don't see any entries for the stuff newly-merged shortly before this PR. Will we need another pass to handle that, or will it come out in the history-wash on release?

Can you be more specific?

**Comment by jahorton** [comment]
> > I don't see any entries for the stuff newly-merged shortly before this PR. Will we need another pass to handle that, or will it come out in the history-wash on release?
> 
> Can you be more specific?

These two PRs, at least, have no entry in history.md despite their inclusion as part of the changeset.
- #11306
- #11302

**Comment by mcdurdin** [comment]
> These two PRs, at least, have no entry in history.md despite their inclusion as part of the changeset.

history.md is updated at release build time. So that will come through overnight.

**Comment by jahorton** [comment]
> > These two PRs, at least, have no entry in history.md despite their inclusion as part of the changeset.
> 
> history.md is updated at release build time. So that will come through overnight.

So, they'll end up as part of... stable-17.0's history.md, I guess?  I forget if/when we copy that over to master's history.md - that's what's on my mind with this line of discussion.

**Comment by mcdurdin** [comment]
They'll be copied over from beta to master when we do one final merge of beta into master. Or we may do it manually if that's the only change.

**Comment by jahorton** [comment]
> They'll be copied over from beta to master when we do one final merge of beta into master.

Ah, see, that's what I thought _this_ was.  My mistake, I suppose.

**Pull request merged**